spec/lib/miq_automation_engine/engine/miq_ae_workspace_runtime_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe MiqAeEngine::MiqAeWorkspaceRuntime do
+  let(:root_tenant) { Tenant.seed }
+  before do
+    @user = FactoryGirl.create(:user_with_group)
```

Comment:
Avoid `@` variables if you can.
Here, you could use a let, or just put the variable directly into the test.